### DISCUSSION TITLE
"onyx" cask is outdated

### DISCUSSION THREAD

**Discussion by pinpontitit** [Homebrew/discussions#4051] [Tap maintenance and Homebrew development]
Hi,
I don't know where I should post to warn about outdated formulae in [formulae.brew.sh](https://formulae.brew.sh/cask/onyx#default).
I don't even know if someone should warn when something is outdated, or if it's something completely cormal, happening a lot of time, and I should just wait ?

The version number(4.2.6) and the system OS requirement(Mac 12) of ["onyx" cask ](https://formulae.brew.sh/cask/onyx#default)are those of the previous release, but when you "brew install" it, it installs the correct current release (4.3.3, Mac 13).

Thanks

**Comment by chenrui333** [comment]
That is actually [a known issue](https://github.com/Homebrew/discussions/discussions/4000), the search index is outdated.

**Reply by SMillerDev** [reply]
No it isn't, this is covered in https://github.com/Homebrew/formulae.brew.sh/issues/566. The version there depends on what device it was generated on.
